src/index.test.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import {
+  render,
+  screen,
+} from '@testing-library/react';
+
+import '../test/jest/__mock__';
+
+import { BrowserRouter as Router } from 'react-router-dom';
```

Comment:
Could you please add new line after  import { BrowserRouter as Router } from 'react-router-dom';?